fixtures/nesting/src/modern/AboutPage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {useContext} from 'react';
+import {findDOMNode} from 'react-dom';
+
+import ThemeContext from './shared/ThemeContext';
+import lazyLegacyRoot from './lazyLegacyRoot';
+
+// Lazy-load a component from the bundle using legacy React.
+const Greeting = lazyLegacyRoot(() => import('../legacy/Greeting'));
+
+export default function AboutPage() {
+  findDOMNode();
```

Comment:
What's this about? Doesn't this warn in all versions of React if called in render?

Reply:
Oops this is an accident.